commit f995cbbd9b052eabc4ebb7c5079885ede0731039
Author: Bogdano Arendartchuk <bogdano@mandriva.com.br>
Date:   Fri Sep 16 15:34:19 2011 -0300

    Always encode the generated changelog in utf-8

diff --git a/RepSys/log.py b/RepSys/log.py
index 57dd141..fbee54b 100644
--- a/RepSys/log.py
+++ b/RepSys/log.py
@@ -20,6 +20,7 @@ import tempfile
 import shutil
 import subprocess
 
+TARGET_ENCODING = "utf8"
 
 locale.setlocale(locale.LC_ALL, "C")
 
@@ -336,8 +337,9 @@ def dump_file(releases, currentlog=None, template=None):
                              "releases" : releases,
                              "revisions_by_author": revisions_author}]
     t = Template(**params)
-    return t.respond()
-
+    su = t.respond()
+    data = su.encode(TARGET_ENCODING)
+    return data
 
 class InvalidEntryError(Exception):
     pass
