commit e7b276e610b08125376ec7975126657e0d13cb46
Author: Bogdano Arendartchuk <bogdano@mandriva.com.br>
Date:   Thu Feb 9 21:44:09 2012 -0200

    --nodeps currently is mandatory to unpack

diff --git a/RepSys/simplerpm.py b/RepSys/simplerpm.py
index 7011185..fb48980 100644
--- a/RepSys/simplerpm.py
+++ b/RepSys/simplerpm.py
@@ -15,7 +15,7 @@ class SRPM:
             self.epoch = None
 
     def unpack(self, topdir):
-        args = ["rpm", "-i", "--define", "_topdir %s" % (topdir),
+        args = ["rpm", "-i", "--nodeps", "--define", "_topdir %s" % (topdir),
                 self.filename]
         execcmd(args)
 
