diff --git a/RepSys/util.py b/RepSys/util.py
index cc3fecb..8e48f93 100644
--- a/RepSys/util.py
+++ b/RepSys/util.py
@@ -75,8 +75,7 @@ def execcmd(cmd_args_or_str, show=False, collecterr=False, cleanerr=False,
     if (not noerror) and proc.returncode != 0:
         if cleanerr:
             msg = output
-        else:
-            cmdline = subprocess.list2cmdline(cmdargs)
+        cmdline = subprocess.list2cmdline(cmdargs)
         raise CommandError(cmdline, proc.returncode, output)
 
     return proc.returncode, output
